extensions/github-authentication/src/githubServer.ts

```diff
@@ -233,7 +233,7 @@ export class GitHubServer implements IGitHubServer {
 			try {
 				const json = await result.json();
 				this._logger.info('Got account info!');
-				return { id: json.id, accountName: json.login };
+				return { id: json.id, accountName: json.login, avatar: json.avatar_url };
```

Comment:
Does this come back when we ask for `user:email` scope or do we need it to have `read:user`?

Reply:
Based on the [GitHub docs](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user): "If the authenticated user is authenticated through OAuth without the user scope, then the response lists only public profile information."
Since a users profile picture is publicly available information. I expect that we can get it just with `user:email`.

To be sure. I just changed from source that we do not request `read:user`, and indeed I got back the avatar nicely.